cmd/operator-sdk/build/cmd.go

```diff
@@ -61,6 +63,8 @@ For example:
 		"Tool to build OCI images. One of: [docker, podman, buildah]")
 	buildCmd.Flags().StringVar(&goBuildArgs, "go-build-args", "",
 		"Extra Go build arguments as one string such as \"-ldflags -X=main.xyz=abc\"")
+	buildCmd.Flags().BoolVar(&skipImage, "skip-image", false,
+		"If set as true, only the operator binary is built and the container image build is skipped.")
```

Comment:
It will skip when run --skip-image only. So, in POV it has more one nit:
```suggestion
		"If set, only the operator binary is built and the container image build is skipped.")
```

Reply:
See: 
<img width="1465" alt="Screenshot 2020-02-24 at 19 47 18" src="https://user-images.githubusercontent.com/7708031/75185731-82119280-573e-11ea-8d0d-7504e8b16848.png">